chapters/statements.tex

```diff
@@ -232,7 +233,7 @@ \subsubsection{Implicit Iteration Ranges}\label{implicit-iteration-ranges}
 \end{lstlisting}
 \end{example}
 
-The size of an array -- the iteration range -- is evaluated on entry to the \lstinline!for!-loop and the array size shall not change during the execution of the \lstinline!for!-loop.
+The size of an array -- the iteration range -- can be evaluated on entry to the \lstinline!for!-loop, since the array size cannot change during the execution of the \lstinline!for!-loop.
```

Comment:
It is a good way to formulate it, but in its current location it looks more like a comment to the example than a clarification of the normative text.  Can we put it in `nonnormative` either right after both normative sections, or in between them?

Reply:
Ok. Done, but I might have messed up some things. Will ensure that it is merged correctly.